test/functional/wallet_sendall.py

```diff
@@ -331,6 +335,9 @@ def sendall_fails_with_transaction_too_large(self):
                 self.wallet.sendall,
                 recipients=[self.remainder_target])
 
+        # Reset the wallet to default
+        self.wallet.keypoolrefill(0)
```

Comment:
Why is `0` the default and what would the call of this RPC do anyway, considering that a full keypool isn't thrown away by calling this.

Reply:
At this point in the test, `self.wallet.getwalletinfo()['keypoolsize']` reports 0.

My concern was that any future tests added after this would incur a slight time penalty when calling `getnewaddress` on the empty keypool (~60ms slower each time for `--legacy-wallet`).

So my intention was to repopulate the keypool with what I thought was the default, but I misread the rpc docs. If I make the correct call (`self.wallet.keypoolrefill()` w/no arguments), the current config results in a new keypool size of 1. Not so helpful.

But this all strikes me now as unnecessary premature optimization.

Can either:
1. Not worry about future tests.
2. Slightly increase the first `keypoolrefill` call to have some leftover (+10? +100?).

Either way, my second `keypoolrefill` that you point out here is unnecessary and will be removed.